HacktoberfestProject.Web/Startup.cs

```diff
@@ -8,6 +8,7 @@
 using HacktoberfestProject.Web.Data.Configuration;
 using HacktoberfestProject.Web.Data;
 using HacktoberfestProject.Web.Services;
+using Microsoft.AspNetCore.HttpOverrides;
```

Comment:
What's this using for?

Reply:
Eeesh we might need to redo this then. I wanted an update of my fork with the latest changes from your repo so I submitted a PR to my fork from yours and merged that first. 
It appears to have included a load of stuff nothing to do with my changes :(

All I changed was adding in a space